src/OSPSuite.SimModel/managed/include/SimModelManaged/ExceptionHelper.h

```diff
@@ -13,6 +13,7 @@ namespace SimModelNET
 		static void ThrowExceptionFrom(const std::string message);
 		static void ThrowExceptionFromUnknown(std::string source);
 		static void ThrowExceptionFromUnknown();
+		static void ThrowExceptionFrom(System::Exception^ ex);
 	};
```

Comment:
New helper function which enables showing call stack of the exception better (especially for the tests)